Microsoft.Azure.CosmosRepository/src/Dtos/LimitedResultRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Azure.CosmosRepository.Dtos
```

Comment:
```suggestion
﻿namespace Microsoft.Azure.CosmosRepository.Pagination
```

Also, need to add the copyright header.